torchgeo/models/ltae.py

```diff
@@ -0,0 +1,213 @@
+"""
+Lightweight Temporal Attention Encoder module
+
+Credits:
+The module is heavily inspired by the works of Vaswani et al. on self-attention and their pytorch implementation of
+the Transformer served as code base for the present script.
+
+paper: https://arxiv.org/abs/1706.03762
+code: github.com/jadore801120/attention-is-all-you-need-pytorch
```

Comment:
Did you mean to reference https://github.com/VSainteuf/lightweight-temporal-attention-pytorch? Also need to add "Copyright (c) 2020 VSainteuf (Vivien Sainte Fare Garnot)" since we are copying code from another library.